eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -20,6 +20,7 @@ parameters:
   PRLabels: ''
   SkipCheckingForChanges: false
   CloseAfterOpenForTesting: false
+  SkipCheckout: false
```

Comment:
Why do you need to SkipCheckout? You can always use a different branch between doing the work and what branch you want to push.